### DISCUSSION TITLE
Livewire and Alpinejs modal error and page freeze

### DISCUSSION THREAD

**Discussion by redlik** [livewire/livewire#5074] [1. Help] [answered]
I'm having a strange issue when closing down a modal made up of alpinejs x-show and livewire saving a new item.
When my save function goes through all the operations on of them is switching showModal to false and this throws these errors and freeze the page:
Uncaught (in promise) Object { isFromCancelledTransition: true }
Uncaught (in promise) TypeError: l is not a function - this one is thrown 2x

The modal opens no problem, close button closes it no problem - only when I send the variable from Livewire it crashes.

**Comment by Martleby** [comment]
Are you using @entangle to share the property between Livewire and Alpine?

**Comment by redlik** [comment, marked as answer]
Hello, Yes I do. 
I think I got it fixed. It turned out I had two instances on Alpinejs, one inside node modules folder and another inside my head section. Removing head link to CDN fixed the issue.
Not sure why it only popped out now. I have other modals on this application and they work fine.

**Reply by protoapostoli** [reply]
Spent HOURS trying to debug why the heck WireUI was doing this, and your answer solved it. Thanks for saving me several more hours of work!

**Reply by yabdab** [reply]
To be clear, was this because you had alpine.js loaded twice on same page?

**Reply by protoapostoli** [reply]
For me it was. I somehow had it loading Alpine twice.

**Reply by AntoineGeorgiev** [reply]
Worked for me as well! Thanks mate!

**Reply by ce-santa** [reply]
Thank you!

**Reply by mgeorgi** [reply]
Thank you soooo much ... I was going nuts over that same error =D

**Reply by shahsani** [reply]
Yes, the solution you mentioned worked. I forgot to remove the Apline include code from app.js... Thank you!

**Reply by drdan18** [reply]
Thank you.   Helped me too.

**Reply by porfinfunciona2** [reply]
Thanks so much, works for me too <3

**Comment by MariaMozgunova** [comment]
I did not have multiple instances of Alpinejs, but updating the Alpinejs version fixed the issue.
The update was from version 3.0.6 to 3.14.8.

**Reply by fhcoding** [reply]
I came across the same issue but removing x-data from the  element and placing it on a parent div solve the error. 
```html
<div x-data="data" x-show="open" x-transition style="display: none">
  <button @click="open=!open">OPEN ME</button>
  <span x-text="message"></span>
</div>   <-- Error from AlpineJs when closing the modal -->

<-- This works perfectly fine  -->
<div x-data="data">
 <div x-show="open" x-transition style="display: none">
  <button @click="open=!open">OPEN ME</button>
  <span x-text="message"></span>
</div> 
</div>
```
